components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -122,8 +129,7 @@ public void checkImage() throws FormatException, IOException {
 
   @Test(dataProvider = "suffixes")
   public void testDefault(String suffix) throws FormatException, IOException {
-    outFile = File.createTempFile("test", suffix);
-    outFile.delete();
+    outFile = tempDir.resolve("test." + suffix).toFile();
```

Comment:
Extra dot in `"test."` (`suffix` already has a leading dot). Does not prevent the test from working fine though, so this change is not required.